webapp/src/components/ActivityPage/Transaction/Transaction.js

```diff
@@ -270,7 +272,7 @@ export default class Transaction extends React.PureComponent {
       case CREATE_ESTATE_SUCCESS: {
         const { estate } = payload
         return t_html('transaction.create_estate', {
-          parcels_link: this.renderParcelsLink(estate.data.parcels)
+          name: estate.data.name
```

Comment:
I think the method `renderParcelsLink` is not using anymore, we can delete it

Reply:
✅ done (faf674f6f69bb2f92c877179f8e50e5396d7efbf)